Makefile

```diff
@@ -460,7 +460,7 @@ PACKAGEMAKER ?= /Developer/Applications/Utilities/PackageMaker.app/Contents/MacO
 PKGDIR=out/dist-osx
 
 release-only:
-	@if `grep -q REPLACEME doc/api/*.md`; then \
+	@if [ "$(DISTTYPE)" = "release" ] && [ `grep -q REPLACEME doc/api/*.md` ]; then \
```

Comment:
Happy to change -- I agree with opt-out but please elaborate on a use case when custom would benefit from the check!